srcalc/src/main/java/gov/va/med/srcalc/util/RetrievalEnum.java

```diff
@@ -1,77 +1,131 @@
 package gov.va.med.srcalc.util;
```

Comment:
You know, this really shouldn't be in the `util` package: that's for general-purpose utilities, not for applications-specific code. I know it has been here for a while, but let's go ahead and move it somewhere more fitting.